### DISCUSSION TITLE
Listen to events

### DISCUSSION THREAD

**Discussion by ari-party** [rigwild/discord-self-bot-console#28] [Q&A] [answered]
Is it possible to listen to events such as "messageCreate"?

**Comment by rigwild** [comment, marked as answer]
Nope. See [Listen to events, do some advanced stuff](https://github.com/rigwild/discord-self-bot-console#listen-to-events-do-some-advanced-stuff)

**Reply by ari-party** [reply]
My main purpose was looking for a message that contains (...) and make a beep sound

**Reply by rigwild** [reply]
See and mark as answer https://github.com/rigwild/discord-self-bot-console/discussions/29#discussion-4288612
